.drone.yml

```diff
@@ -1,32 +1,55 @@
+---
 kind: pipeline
-type: docker
+type: kubernetes
 name: default
 
-
 steps:
   - name: test
-    image: golang:1.13
+    image: golang:1.13-buster
     commands:
-      - go test ./cmd/... ./internal/...
-      - go vet ./cmd/... ./internal/...
+      - make test
+
   - name: lint
     image: cytopia/golint
     commands:
-      - golint -set_exit_status ./cmd/... ./internal/...
-  - name: build
-    image: golang:1.13
-    commands:
-      - go build -o build/drone-helm cmd/drone-helm/main.go
+      - golint -set_exit_status ./...
```

Comment:
Let's combine these and use golangci-lint as we have started doing in other repos.

```
  - name: test
    image: golangci/golangci-lint:v1.30.0-alpine
    commands:
      - apk update
      - apk add make
      - make test
```

I believe this version uses Go 1.15, so you'll want to adjust accordingly.

Reply:
the pipeline is now using the golangci image